### PR TITLE
Add FreeBSD CI

### PR DESCRIPTION
**Pull request by Zoid-dr**

### CODE REVIEW

**Comment by aleixpol** [comment]
@arrowd can you review it maybe?

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+task:
+  freebsd_instance:
+    matrix:
+      - image_family: freebsd-13-2-snap
+  env:
+    LOCALBASE: /usr/local
+    LDFLAGS: -L$LOCALBASE/lib
```
Comment (arrowd) [inline .cirrus.yml:7]: Is this really needed?

Comment (Zoid-dr) [reply .cirrus.yml]: No. Fix.

**Comment by arrowd** [comment]
Now the whole `env` block isn't needed too.

**Comment by arrowd** [comment]
Looks good to me.

Tests are omitted for now, as one of them is failing and requires some debugging.

**Comment by arrowd** [comment]
Can we get this in? I'm working on a FreeBSD backend and it is important to us to be sure the new code keeps working on FreeBSD.

**Comment by Conan-Kudo** [comment]
I just noticed this after merging it, but please don't add `Sponsored by` information in PackageKit commits. I do not appreciate abusing revision control for permanent advertising of companies.

**Comment by arrowd** [comment]
Ok, noted. This isn't a hard requirement for us.

**Comment by Conan-Kudo** [comment]
@arrowd I've activated Cirrus CI on the project and it seems to be running. Is there a way to make it so that the name of the job isn't just "main" but something more useful (like have "freebsd" in the name or something)?

**Comment by arrowd** [comment]
Sure, here is how this is done: https://github.com/CollaboraOnline/online/blob/master/.cirrus.yml#L2

**Comment by Conan-Kudo** [comment]
Thanks, this is all set up now! Thanks for the contribution!

**Pull request merged**